corehq/apps/celery/__init__.py

```diff
@@ -18,3 +18,25 @@ def _init_celery_app():
     app.config_from_object('django.conf:settings', namespace='CELERY')
     app.autodiscover_tasks()
     app.set_default()
+
+
+class PeriodicTask(Task):
+
+    @classmethod
+    def on_bound(cls, app):
+        app.conf.beat_schedule[cls.name] = {
+            'task': cls.name,
+            'schedule': cls.run_every,
+            'args': (),
+            'kwargs': {},
+            'options': cls.options or {}
+        }
+
+
+def periodic_task(*args, **options):
```

Comment:
Why `*args`? It doesn't appear to be used.

Reply:
Yeah. Only `**options` are used. I kept `*args` to make it consistent with the implementation that celery used to have https://github.com/celery/celery/blob/b2668607c909c61becd151905b4525190c19ff4a/celery/task/base.py#L275 

Since we are not using it, I think it would be a good idea to remove it.

Reply:
https://github.com/dimagi/commcare-hq/pull/31584/commits/0403bc088115d41c7eb35e3a0143808c48a1218c